### PR TITLE
Python 2to3

### PR DESCRIPTION
**Pull request by satreix**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,4 +14,4 @@ branches:
     - master
 
 script:
-  - "pip install pygments_github_lexers"
+  - "pip install git+https://github.com/satreix/pygments-github-lexers@master"
```
Comment (liluo) [inline .travis.yml]: remove this line?

Comment (satreix) [inline .travis.yml]: Will do :+1: 
It was just to make Travis pull my repo for tests.

**Comment by satreix** [comment]
Remove the edit in .travis.yml (that why the build fails -_-)

**Comment by liluo** [comment]
@satreix I will release new verstion, just minutes.

**Comment by liluo** [comment]
@satreix 

Thank you for pull request!!!

pygments-github-lexers v0.0.4 is released:  https://pypi.python.org/pypi/pygments-github-lexers
And CI is passing:  https://travis-ci.org/liluo/pygments-github-lexers

**Comment by satreix** [comment]
You are welcome. I'm really looking forward to use your version of linguist in python3.
Can you check out this issue please: liluo/mime#2

**Pull request merged**